thousandeyes/schemas.go

```diff
@@ -20,6 +20,20 @@ var schemas = map[string]*schema.Schema{
 			},
 		},
 	},
+	"agents--label": {
```

Comment:
single `-` here?

Reply:
I'm open to doing this differently, but there is some intention behind this.  This is a -- to indicate that it is not a part of the field name from the API's perspective.

Fields that have the same name are generally treated the same across tests.  But we have a few tests for which this is not the case.  My first thought was to just change these within the resource files for tests that are the exceptions.  But there are a couple of cases where more than one test uses the alternate behavior, so it might make sense to keep those together with other schemas.  

I also thought it might be beneficial to see the alternate definitions beside the default to make it more clear that the behavior is not always the same.  

But on reflection I agree that it isn't clear enough.  Options here I see are to add a comment here, put these into the different resource files, or change the separator.  Let me know what you prefer and I'll make a commit to do just that.

Reply:
Based on the above, how would you like me to address this?

Reply:
Sorry been sleeping on this as I moved companies/roles. Will try to take a look tomorrow. But I would say everything should follow the standard syntax of _ between words. I'm not keen on the double dash.

Reply:
No worries, we're comfortable using branches here until then.  I hope your new position is treating you well!

It can definitely be done differently.  It was about delineating between separation between words and separation between identifiers.  But I'm not advocating a particular way of solving it, just noting the difference.  If we end up not distinguishing between the two, nothing breaks.

Reply:
Hi Sorry for the huge delay on this. If you update to single `-` then im happy to get this merged.